frontend/utils/transitions.tsx

```diff
@@ -0,0 +1,12 @@
+import { Slide } from '@mui/material'
+import { TransitionProps } from '@mui/material/transitions'
+import { forwardRef } from 'react'
+
+export const Transition = forwardRef(function Transition(
+  props: TransitionProps & {
+    children: React.ReactElement<any, any>
+  },
+  ref: React.Ref<unknown>,
+) {
+  return <Slide direction={'up'} ref={ref} {...props} />
```

Comment:
It would be nice if we could provide a direction as a prop, just in case we want to use different directions in certain situations.

We should leave `up` as the default though, that way we don't need to alter any of your other changes.

Reply:
I don't know if I agree. The point of using the same transition component across the board was to keep everything feeling consistent. If we start allowing the direction to be customisable, we are moving away from the aim of being consistent.

Reply:
I'm convinced by @ARADDCC012's comment, @IR96334 feel free to disregard my comment!